app/helpers/applications_helper.rb

```diff
@@ -34,11 +34,20 @@ def formatted_application_data_value(key, value)
       project = Project.find_by_id value
       link_to_if project, project.try(:name), project
     else
-      auto_link simple_format(value)
+      auto_link(simple_format(value))
     end
     content_tag :p, formatted.html_safe
   end
 
+  def show_student_names(application)
+    output = ""
+    output << content_tag(:b, Application.data_label("student0_name"))
+    output << content_tag(:p, simple_format(application.team.students.first.name))
```

Comment:
yes, you are right, I just copy & pasted from the current implementation. it's fixed in the unrolled partial.